FRW/Services/Token/TokenBalanceProvider.swift

```diff
@@ -14,16 +14,77 @@ protocol TokenBalanceProvider {
     func getFTBalance(address: FWAddress) async throws -> [TokenModel]
     func getFTBalanceWithId(address: FWAddress, tokenId: String) async throws -> TokenModel?
     func getNFTCollections(address: FWAddress) async throws -> [NFTCollection]
+    func getAllNFTs(address: FWAddress) async throws -> [NFTModel]
```

Comment:
For now, we need get all NFTs under a specific collection, not all collection. For example, get all NBA ts NFT under this account. Not all nfts under this account

Reply:
Isn't that what `func getNFTCollectionDetail(address: FWAddress, collectionIdentifier: String, offset: Int) async throws -> NFTListResponse` does?